scripts/run.sh

```diff
@@ -177,14 +178,18 @@ mkdir -p ${RECO_TEMP}
 
 # Data egress to directory
 if [ "${COPYFULL:-false}" == "true" ] ; then
-  # Token for write authentication
-  export BEARER_TOKEN=$(cat ${_CONDOR_CREDS}/eic.use)
-  if [ -n ${XRDWURL} ] ; then
-    xrdfs ${XRDWURL} mkdir -p ${XRDWBASE}/${FULL_DIR} || echo "Cannot write simulation outputs to xrootd server"
+  if [ "${USERUCIO:-false}" == "true" ] ; then
+    python /opt/campaigns/hepmc3/scripts/register_to_rucio.py -f "${FULL_TEMP}/${TASKNAME}.edm4hep.root" -d "/${FULL_DIR}/${TASKNAME}.edm4hep.root" -s epic
   else
-    mkdir -p ${XRDWBASE}/${FULL_DIR} || echo "Cannot write simulation outputs to xrootd server"
+    # Token for write authentication
+    export BEARER_TOKEN=$(cat ${_CONDOR_CREDS}/eic.use)
```

Comment:
`set -Euo pipefail` will cause failure when `_CONDOR_CREDS` is unset (presumably on slurm systems).

Reply:
How do you propose to get around it?

Reply:
```suggestion
    export BEARER_TOKEN=$(cat ${_CONDOR_CREDS:-.}/eic.use)
```

Reply:
i.e. default to $PWD, and other systems can figure out how to pass `eic.use` that way (in slurm: by placing the use file in your local job directory somehow).

Reply:
In general, it may be useful to use shellcheck on these scripts as part of CI.